vars/runTest.groovy

```diff
@@ -117,19 +117,31 @@ def call(Map config = [:]) {
     // https://issues.jenkins-ci.org/browse/JENKINS-39203
     // Once that is fixed all of the below should be pushed up into the
     // Jenkinsfile post { stable/unstable/failure/etc. }
+    
+    // Somehow we now have two different names for storing the expected
+    // junit result file names.
```

Comment:
Where would this be coming from?  _testResults_ is not even a documented parameter.  Probably whatever caller is using that parameter rather than _junit_files_ needs to be fixed.  We shouldn't add hacks here for callers using the wrong parameter.